query-utils.js

```diff
@@ -350,3 +349,12 @@ export async function isCKB(eenheidUri) {
 
   return (await querySudo(queryStr)).boolean;
 }
+
+export async function isDecidableByCKB(forDecisionType) {
```

Comment:
I am a bit reluctant here to add an extra dependency on business rules; if these change, yet another place to think about changing here too. 

I am wondering whether, for this service and this function, we can have two modes for this service. Worship-submissions-db vs 'normal-mode'. Would this make fetching document details more simple?

Reply:
The flow is still complex, but the code is easier to understand now :+1:

Reply:
Alright I'll move on with this version, hopefully if something is still off it'll be caught during the testing phase. Thanks for checking it out!